src/components/Navbar/SettingsDrawer/Section/Row.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 
 import styles from './Row.module.scss';
 
-const Row = ({ children }) => <div className={styles.row}>{children}</div>;
+interface RowProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+const Row = (props: RowProps) => <div className={styles.row} {...props} />;
```

Comment:
I prefer to explicitly show which props we are passing exactly since it's very vague looking at the component and we have to go check the usage to know what was passed etc.